pkgs/applications/misc/electrum-ltc/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv
+, fetchurl
+, python2Packages
+}:
+
+python2Packages.buildPythonApplication rec {
+  name = "electrum-ltc-${version}";
+  version = "2.6.4.2";
+
+  src = fetchurl {
+    url = "https://electrum-ltc.org/download/Electrum-LTC-${version}.tar.gz";
+    sha256 = "0sqcyk6n6kgaiinnwh6mzbbn4whk3ga59r5bw5rqmnnfqk1xdnb4";
+  };
+
+  propagatedBuildInputs = with python2Packages; [
+    pyqt4
+    slowaes
+    ecdsa
+    pbkdf2
+    requests
+    qrcode
+    ltc_scrypt
+    protobuf3_0
+    dns
+    jsonrpclib
+  ];
+
+  preBuild = ''
+    sed -i 's,usr_share = .*,usr_share = "'$out'/share",g' setup.py
+    pyrcc4 icons.qrc -o gui/qt/icons_rc.py
+    # Recording the creation timestamps introduces indeterminism to the build
+    sed -i '/Created: .*/d' gui/qt/icons_rc.py
+  '';
+
+  doInstallCheck = true;
```

Comment:
`buildPythonApplication` runs the `installCheckPhase` by default, but we just call it `installCheckPhase`. So here you can skip the `doInstallCheck` line and write `checkPhase` instead of `installCheckPhase`.